.github/workflows/tests.yml

```diff
@@ -38,8 +65,20 @@ jobs:
             /bin/bash uninstall.sh -f >/dev/null
           fi
 
+      - name: Set up WSL environment
+        if: runner.os == 'windows'
+        shell: "wsl-bash -u root {0}"
+        run: |
+          echo "runner ALL=(ALL) NOPASSWD:ALL" | tee /etc/sudoers.d/runner
+          chmod 644 /etc/sudoers.d/runner
+          echo -e "#!/bin/bash\nexec /home/linuxbrew/.linuxbrew/bin/brew \"\$@\"" | tee /usr/local/bin/brew
+          chmod 755 /usr/local/bin/brew
+          apt-get install build-essential -y
```

Comment:
You can let the action do that right away if you like wiht `additional-packages` input. :-)

Reply:
And otherwise you either need to do `update: true` for the action or manually do `apt-get update` before the install. :-)